accessibility/accessibility-in-pdfs.md

```diff
@@ -0,0 +1,92 @@
+# Accessibility in PDFs
+
+## A warning
+
+If you’re concerned about the accessibility of your PDF documents, then you should [consult a specialist like AbilityNet](https://www.abilitynet.org.uk/creating-accessible-pdfs) to review what you have, make expert recommendations, and potentially remediate issues for you. 
```

Comment:
Probably best to expand the acronym e.g. "Portable Document Format (PDF)" the first time we use it?